### PR TITLE
updates to support solr based search

### PR DESCRIPTION
**Pull request by fdefalco**

### CODE REVIEW

File: js/pages/vocabulary/components/search.js [hunk 1]
```diff
@@ -30,330 +30,331 @@ define([
...
-			this.isInProgress = ko.computed(() => {
-				return this.domainsLoading() === true && this.vocabulariesLoading() === true;
-			});
-			this.isInProgress.subscribe((isInProgress) => {
-				if (!isInProgress) {
-					if (params.query()) {
+				this.isInProgress = ko.computed(() => {
+					return this.domainsLoading() === true && this.vocabulariesLoading() === true;
+				});
+				/*
```
Comment (pavgra) [inline js/pages/vocabulary/components/search.js]: Why not to remove this if not required?

File: js/pages/vocabulary/components/search.js [hunk 2]
```diff
@@ -30,330 +30,331 @@ define([
...
-				vocabularies: new Set(),
-			};
-			this.data = ko.observableArray([]);
-			this.searchExecuted = ko.observable(false);
-			this.searchColumns = ko.observableArray([]);
-			this.searchOptions = ko.observable();
+		class Search extends AutoBind(Component) {
+			constructor(params) {
+				super(params);
+				console.log(params);
```
Comment (pavgra) [inline js/pages/vocabulary/components/search.js]: Is this required?

File: js/pages/vocabulary/components/search.js (hunk 1 above)
Comment (fdefalco) [reply js/pages/vocabulary/components/search.js]: Is it not required?  This wasn't a change in this PR.

**Comment by anthonysena** [comment]
Noting that this is related to OHDSI/WebAPI#580

**Pull request merged**